.github/workflows/python-ec2-default-test.yml

```
@@ -171,7 +176,7 @@ jobs:
               -var="get_adot_wheel_command=${{ env.GET_ADOT_WHEEL_COMMAND }}" \
               -var="language_version=${{ env.PYTHON_VERSION }}" \
               -var="cpu_architecture=${{ env.CPU_ARCHITECTURE }}" \
-            || deployment_failed=$?
+              -var="custom_metrics_enabled=${{ env.CUSTOM_METRICS_ENABLED }}" \
```

Keep the old line too
```suggestion
              -var="custom_metrics_enabled=${{ env.CUSTOM_METRICS_ENABLED }}" \
              || deployment_failed=$?
```